### PR TITLE
Feature/enable dominant speaker changed event in API

### LINKED ISSUE #4049: Dominant speaker recognition in API
I am trying to print to screen the dominant speaker whenever it changes to improve accesibility for disabled people.

It would be very interesting to have an event like DOMINANT_SPEAKER_CHANGED in the lib, but for the jitsi-meet API.

I tried to get it working with the lib but when I implement the lib it's broken.

Thank you all for your work.

### PR DESCRIPTION
**Pull request by karthikmuralidharan**
Closes #4049

### CODE REVIEW

File: conference.js [hunk 1]
```diff
@@ -1820,7 +1820,11 @@ export default {
 
         room.on(
             JitsiConferenceEvents.DOMINANT_SPEAKER_CHANGED,
-            id => APP.store.dispatch(dominantSpeakerChanged(id, room)));
+            id => {
+                APP.store.dispatch(dominantSpeakerChanged(id, room));
+                APP.API.notifyDominantSpeakerChanged(id);
```
Comment (saghul) [inline conference.js]: Can you please move the call to APP.API here? https://github.com/jitsi/jitsi-meet/blob/master/react/features/external-api/middleware.js

Comment (virtuacoplenny) [reply conference.js]: Hooking onto "DOMINANT_SPEAKER_CHANGED" might be easiest. So somewhere in one of the switch statements:
```
import {
    ....
    DOMINANT_SPEAKER_CHANGED,
    ....
} from '../base/participants';
...
case DOMINANT_SPEAKER_CHANGED:
    APP.API.notifyDominantSpeakerChanged(action.participant.id);
    break;
```

Comment (karthikmuralidharan) [reply conference.js]: Yep, will do. Thanks!!

**Comment by karthikmuralidharan** [comment]
@saghul I've addressed the review comments as per your recommendation.

File: react/features/external-api/middleware.js [hunk 2]
```diff
@@ -163,6 +164,10 @@ MiddlewareRegistry.register(store => next => action => {
     case SUBMIT_FEEDBACK_SUCCESS:
         APP.API.notifyFeedbackSubmitted();
         break;
+
+    case DOMINANT_SPEAKER_CHANGED:
```
Comment (saghul) [inline react/features/external-api/middleware.js]: can you please sort the `case` statement alphabetically?

Comment (karthikmuralidharan) [reply react/features/external-api/middleware.js]: Done

**Comment by saghul** [comment]
jenkins please test this please

**Comment by karthikmuralidharan** [comment]
@saghul something wrong with the CI ? 

I see 404 pages on clicking on the details :

http://pr-reports.jitsi.org/jitsi-meet-PR-test/6552

**Pull request merged**